### DISCUSSION TITLE
part-ii/2017-77/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#168] [Question comments]
Part II 2017 Graph Theory: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2017-77/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part is bookwork.

An example that is Hamiltonian is $K_4$, and an examples that is not Hamiltonian is $K_4 + x$, where $x$ is a new vertex adjacent to one of the vertices of the complete graph (this is not Hamiltonian because every vertex that appears in a cycle must have degree at least $2$).

A graph of this form can be built up by starting with the smallest cycle in $G$, which has $\chi(G) \leq 3$, then successively replacing edges with two edges and a vertex in between (this does not cause us to use more than $3$ colours, because $1$ of the $3$ colours will not appear in the endpoints of the edge we got rid of).

</details>
